src/target/lpc55xx.c

```diff
@@ -37,12 +37,478 @@
 #include "target_internal.h"
 #include "cortexm.h"
 
-#define LPC55xx_CHIPID 0x50000ff8U
-#define LPC55_DMAP_IDR 0x002A0000U
+#define IAP_FREQ_MHZ 96
+#define ARM_BKPT_PATTERN 0xBEBE
 
+#define LPC55_DMAP_IDR                 0x002A0000
 #define LPC55_DMAP_BULK_ERASE          0x02U
 #define LPC55_DMAP_START_DEBUG_SESSION 0x07U
 
+#define LPC55XX_FLASH_BASE     0x00000000
```

Comment:
U on the ends of all these constants please, clang-format, and use `LPC55xx_` in the macro naming scheme so it's more obvious the X's refer to missing part number digits